src/main/java/com/google/sps/servlets/ImageFormServlet.java

```diff
@@ -66,6 +65,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
             byte[] imageBytes = filePart.getInputStream().readAllBytes(); //
             List<EntityAnnotation> imageLabels = getImageLabels(imageBytes);
 
+            List<Value<String>> comments = new ArrayList<>();
```

Comment:
Where does this get a value? Or do we just stick an empty list in initially?

Reply:
Yeah, it's an empty list initially. I handle if the element is null on the server side.